### DISCUSSION TITLE
Deceptive site ahead

### DISCUSSION THREAD

**Discussion by ghost** [community/community#57690] [Repositories] [answered]
### Select Topic Area

Question

### Body

From yesterday morning, all the projects that i hosted on github pages are showing something like deceptive site ahead.Attackers on nishaa0315.github.io may trick you into doing something dangerous like installing software or revealing your personal information (for example, passwords, phone numbers, or credit cards).
I want to know what is the exact issue and how i can fix it..

**Comment by airtower-luna** [comment, marked as answer]
That sounds like (at least) one of you sites ended up on the [Google Safe Browsing blocklist](https://safebrowsing.google.com/). I notice you have a https://github.com/nishaa0315/Netflix-Clone repository, that'd be a possible reason for the "deceptive site" warning (imitating a well-known site is a common phishing strategy).

See https://developers.google.com/search/docs/monitor-debug/security/social-engineering?hl=en for details on that kind of warning. To fix the problem you'll need to remove any imitation site (leaving the repository should be okay), and then [request a review](https://safebrowsing.google.com/safebrowsing/report_error/) of the report.

**Reply by SaleemNijim** [reply]
i have the same issue with the same project !!

**Reply by yssnprasad** [reply]
i have the same issue with the same project !!
how to solve the issues?

**Comment by imvickykumar999** [comment]
Google Blacklisted my all GitHub Pages, for the same reason.
